CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-## 3.5.1
-
```

Comment:
As part of the last release, I noticed this hanging out up here.  I moved it to the correct spot.  When switching from semantic-release to changesets this happens (happened with other repos too!)

Reply:
TIL